jmix-core/core/src/main/java/io/jmix/core/CoreProperties.java

```diff
@@ -100,6 +100,12 @@ public class CoreProperties {
      */
     PessimisticLock pessimisticLock;
 
+    /**
+     * Whether BigDecimalDatatype, DoubleDatatype and FloatDatatype should round the actual parsed value
+     * according to format settings.
+     */
+    boolean decimalValueRoundByFormat;
```

Comment:
Maybe rename the property to `roundDecimalValueByFormat`? WDYT?

Reply:
Looks a little better - renamed